sensor.ino

```diff
@@ -0,0 +1,31 @@
+#include <Smartcar.h>
+
+
+const int cruiseSpeed = 40;
+const int triggerDist = 200;
+
+const int lDegrees = -75; // degrees to turn left
+
+unsigned short TRIGGER_PIN = 6;
+unsigned short ECHO_PIN = 7;
+const unsigned int MAX_DISTANCE = 1000;
+
+ArduinoRuntime arduinoRuntime;
+BrushedMotor leftMotor{arduinoRuntime, smartcarlib::pins::v2::leftMotorPins};
+BrushedMotor rightMotor{arduinoRuntime, smartcarlib::pins::v2::rightMotorPins};
+DifferentialControl control{leftMotor, rightMotor};
+SimpleCar car{control};
+SR04 sensor(arduinoRuntime, TRIGGER_PIN, ECHO_PIN, MAX_DISTANCE);
+
+void setup() {
+  Serial.begin(9600);
+  car.setSpeed(cruiseSpeed);
+}
+void loop() {
+	if (sensor.getDistance() > 0 && sensor.getDistance() < triggerDist){

```

Comment:
You are calling `getDistance` twice. This means that two separate measurements will be conducted which can be potentially different.
This can cause problems if they are very different, for example, if the first one is `300` and the second `0`.